app/api/sugar.coffee

```diff
@@ -1,34 +1,35 @@
 JSONAPIClient = {Resource} = require 'json-api-client'
 authClient = require './auth'
 config = require './config'
-
 SugarClient = require 'sugar-client'
-SugarClient.Primus = require 'sugar-client/primus'
-SugarClient.host = config.sugarHost
 
-window.sugarApiClient = new JSONAPIClient config.sugarHost,
```

Comment:
Oops, probably.  It's requirable with `require('<whatever>/api/sugar').api`  Same thing with the socket client.